### DISCUSSION TITLE
Periodic boundary conditions in xtb

### DISCUSSION THREAD

**Discussion by Ameneh3526** [grimme-lab/xtb#988] [General]
I try to run the MD simulation and geometry optimization under periodic conditions by using --mcgfnff flag in xtb. After running the program I received this error, 

-1- prog_main_parseArguments: Unknown option '--mcgfnff' provided

And by default, the program is run based on GFN2-xtb.

I don't know what to do to run my program in periodic conditions.
I need to run my program with --mcgfnff flag.

**Comment by Albkat** [comment]
which version of xtb you are using?

**Reply by Ameneh3526** [reply]
I use xtb version 6.6.1

**Reply by Albkat** [reply]
mcGFN-FF is implemented in the xtb version 6.7.0.

**Reply by Thomas3R** [reply]
The periodic boundary conditions for GFN-FF, as well as the `--mcgfnff` flag are only included in the latest version 6.7.0.
You can find the release here: https://github.com/grimme-lab/xtb/releases/tag/v6.7.0

**Reply by Ameneh3526** [reply]
Thank you so much.
